src/me/eglp/gv2/util/base/guild/reminder/ReminderRepetitionEnum.java

```diff
@@ -0,0 +1,49 @@
+package me.eglp.gv2.util.base.guild.reminder;
+
+/**
+ * This is a reminders helper Enum, to signal, how long to wait, between
+ * repetitions of the reminder.
+ * 
+ * @author The Arrayser
+ * @date Mon Mar 27 19:49:22 2023
+ */
+
+public enum ReminderRepetitionEnum {
```

Comment:
Minor issue, but I think the enum should be called `ReminderRepetition`. No need to describe what it is in the name